paymentsheet/src/test/java/com/stripe/android/paymentsheet/IntentConfirmationFlowTest.kt

```diff
@@ -200,7 +200,7 @@ internal class IntentConfirmationFlowTest {
         assertThat(failAction.cause).isInstanceOf(IllegalStateException::class.java)
         assertThat(failAction.cause.message).isEqualTo("An error occurred!")
         assertThat(failAction.message).isEqualTo(R.string.stripe_something_went_wrong.resolvableString)
-        assertThat(failAction.errorType).isEqualTo(PaymentConfirmationErrorType.Internal)
+        assertThat(failAction.errorType).isEqualTo(PaymentConfirmationErrorType.Payment)
```

Comment:
The handler used to return `Payment` for `IntentConfirmationDefinition` which is the more accurate error type to return. Updated the definition to do this as well.